pkgs/by-name/ra/radiance/package.nix

```
@@ -0,0 +1,47 @@
+{
+  cmake,
+  fetchFromGitHub,
+  runCommand,
+  lib,
+  libGLU,
+  stdenv,
+  libX11,
+  tcsh,
+  tk,
+}:
+let
+  csh = runCommand "csh" { } ''
+    mkdir -p $out/bin
+    ln -s ${lib.getExe tcsh} $out/bin/csh
+  '';
+in
+stdenv.mkDerivation {
+  pname = "radiance";
+  version = "5.4";
+
+  src = fetchFromGitHub {
+    owner = "LBNL-ETA";
+    repo = "radiance";
+    rev = "refs/tags/rad5R4";
```

For easier future version updates let's use `version` to construct the `src.rev` as needed.
```suggestion
    rev = "refs/tags/rad${lib.versions.major}R${lib.versions.minor}";
```